include/re_net.h

```diff
@@ -27,11 +27,7 @@
 #endif
 
 /** Length of IPv4/v6 address string */
-#ifdef HAVE_INET6
 #define NET_ADDRSTRLEN INET6_ADDRSTRLEN
-#else
-#define NET_ADDRSTRLEN INET_ADDRSTRLEN
-#endif
 
```

Comment:
It looks like `NET_ADDRSTRLEN` is unused. or am I missing something?

Reply:
We can probably remove it ... it is most likely not used ...